Update Ruleset Engine concepts (#1603)
cloudflare/cloudflare-docs: products/firewall/src/content/cf-rulesets/custom-rules/custom-firewall/create-api.md

## Create a Custom Firewall rule

To create a Custom Firewall rule, add a rule to the `http_request_firewall_custom` phase entry point ruleset.

1. Invoke the [View ruleset](/cf-rulesets/rulesets-api/view#view-a-specific-ruleset) method to obtain the list of rules already present in the `http_request_firewall_custom` phase entry point ruleset. If the entry point ruleset does not exist, proceed to step 2, since adding a rule to the entry point ruleset will create the ruleset if it does not exist.

> 1. Invoke the [Update ruleset](/cf-rulesets/rulesets-api/update) method to update the list of rules in the phase entry point ruleset with a new rule. You must include (at least) the rule ID of all the rules you wish to keep in the ruleset.

Reviewer: ```suggestion
1. Invoke the [Update ruleset](/cf-rulesets/rulesets-api/update) method to update the list of rules in the phase entry point ruleset with a new rule. You must include the rule ID of all the rules you wish to keep in the ruleset.
```

Reviewer: I would remove at least and add something like "..., all other fields are optional" solidifying it as a minimal requirement. Or just leave it out.

> 1. Invoke the [Update ruleset](/cf-rulesets/rulesets-api/update) method to update the list of rules in the phase entry point ruleset with a new rule. You must include (at least) the rule ID of all the rules you wish to keep in the ruleset.

Author: ```suggestion
1. Invoke the [Update ruleset](/cf-rulesets/rulesets-api/update) method to update the list of rules in the phase entry point ruleset with a new rule. You must include the rule ID of all the rules you wish to keep in the ruleset (all other fields are optional).
```

Author: Adding note at the end of the sentence, as suggested.
